lib/utilities/healthcheckHandler.js

```diff
@@ -121,7 +121,8 @@ function healthcheckHandler(clientIP, req, res, log, statsClient) {
         });
     }
 
-    const deep = (req.url === '/ready');
+    // CLDSRV-740: S3C with the healthcheck on s3 port needs the internal route
```

Comment:
- Can we upgrade `s3frontend` to use the new route instead, to avoid introducing config flag and divergence? Seems the preferable approach here...
- If I remember correctly, we already have an option used to "enable" healthcheck on dedicated port (→ `/ready`, `/health`) or on the usual route (`/_/healthcheck`) -maybe something like "healthcheckPort"-. If we need to have separate behaviors, this change use the same setting?

Reply:
the flag is `metricsPort` I think

Reply:
We don't want to change the `metricsPort`. For now we use `metricsPort` only to scrape metrics. And healthcheck is on the s3 port.

> Can we upgrade s3frontend to use the new route instead,

No because with the same port, internal services are prefixed with `/_/<internalServiceName` otherwise it's a regular S3 request. But We could use a name like `/_/healthcheck/ready`